test/e2e/scheduling/predicates.go

```diff
@@ -825,6 +825,7 @@ func CreateHostPortPods(f *framework.Framework, id string, replicas int, expectR
 
 // create pod which using hostport on the specified node according to the nodeSelector
 func createHostPortPodOnNode(f *framework.Framework, podName, ns, hostIP string, port int32, protocol v1.Protocol, nodeSelector map[string]string, expectScheduled bool) {
+	hostIP = framework.IPv4IPv6Translator(hostIP)
```

Comment:
is this the only known test that breaks for ipv6?

Reply:
I´m sure there are more e2e tests that break ipv6, but this is the only one that break conformance e2e tests.
https://testgrid.k8s.io/conformance-kind#kind%20(IPv6),%20master%20(dev)